mne/tests/test_evoked.py

```diff
@@ -452,3 +452,33 @@ def test_array_epochs():
     types = ['eeg'] * 19
     info = create_info(ch_names, sfreq, types)
     assert_raises(ValueError, EvokedArray, data1, info, tmin=-0.01)
+
+
+def test_add_channels():
+    """Test evoked splitting / re-appending channel types
+    """
+    evoked = read_evokeds(fname, condition=0)
+    evoked.info['buffer_size_sec'] = None
```

Comment:
That's a good question. Right now it populates that field when creating any new info object. I had never run into this problem before trying to get this test to pass. Has anyone else had it before?